Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: clean test lint migrate
+
+clean:
+	find . -name "*.pyc" -print0 | xargs -0 rm -rf
+	-rm -rf htmlcov
+	-rm -rf .coverage
+	-rm -rf build
+	-rm -rf dist
+	-rm -rf src/*.egg-info
+
+lint:
+	poetry run black . --line-length=79
```

Comment:
do we want to respect line-length? I find it ugly

Reply:
I feel like we should. It's suggested in PEP 8 and following this would stop unnecessary arguments/discussions and lead to the most important thing which is writing clean, readable code. What line length would you suggest and why?

Reply:
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

Line length at 79 used to be the default choice because old terminals used to support only 80 characters per line, 79 imo is just ugly